features/completed-form-available-to-download.feature

```diff
@@ -0,0 +1,58 @@
+@e2e
+@javascript
+Feature:
+  As an ATOS admin
+  I want to be able to download completed forms
+  So I can triage an employees claim against their employer
+
+  Scenario: PDF format
+    Given I am making a claim
+    When I submit a completed Employment Tribunal form
+    Then the completed form in PDF format is available for ATOS to download and validate
```

Comment:
thinking ahead a little - as we are gonna be talking about a claim form and a respondent form once et3 stuff is included - might be worth just calling everything a claim form now ?